src/moneyTracker.js

```diff
@@ -0,0 +1,35 @@
+function Moneytracker() {
+  this._transactions = [];
+};
+
+Moneytracker.prototype.recordEarning = function (money, date) {
+  this._transactions.push(new Transaction(money, date))
+};
+
+Moneytracker.prototype.recordExpenditure = function (money, date) {
+  this._transactions.push(new Transaction(money, date))
+};
+
+Moneytracker.prototype._balance = function () {
```

Comment:
It feels like `_balance()` has multiple responsibilities - it updates the balance on transactions, and also returns the balance.  Could these be split up?